packages/pigeon/platform_tests/test_plugin/android/src/main/kotlin/com/example/test_plugin/TestPlugin.kt

```diff
@@ -867,3 +872,57 @@ class TestPluginWithSuffix : HostSmallApi {
     callback(Result.success(Unit))
   }
 }
+
+object SendInts : StreamIntsStreamHandler() {
+  val handler = Handler(Looper.getMainLooper())
+
+  override fun onListen(p0: Any?, sink: PigeonEventSink<Long>) {
+    var count: Long = 0
+    val r: Runnable =
+        object : Runnable {
+          override fun run() {
+            handler.post {
+              sink.success(count)
+              count++
+            }
+            handler.postDelayed(this, 1000)
```

Comment:
Is this for an automated test? If so, this is an extremely long delay. (If not, can we comment to say what it's for?)

Reply:
That's just a leftover from some debugging I was doing. Should be much shorter